packages/generator-emakinacee-react/generators/app/templates/shared/dot/.eslintrc

```diff
@@ -1,7 +1,10 @@
 {
     "extends": [
         "emakinacee-base",
-        "emakinacee-react"
+        "emakinacee-react",
+        // Must be last to override rules. https://github.com/prettier/eslint-config-prettier#installation
+        "prettier",
+        "prettier/react"
```

Comment:
Yeah cool, so I think we should go with the easier solution for now.
I will add an issue on the other repo to align our ESLint configs with prettier and maybe we can remove the  the prettier configs afterwards.

Thanks for your work @sladiri